env/agentcompass/tests/test_agentcompass.py

```
@@ -0,0 +1,963 @@
+        "session_id": "session-1",
+        "agent_name": "agentcompass",
+        "agent_id": "row-1",
+        "group_id": "group-1",
+        "gateway_base_url": "http://127.0.0.1:8000/v1/sessions",
+        "model": "test-route",
+        "temperature": 0.25,
+        "max_steps": 7,
+        "storage_type": "sqlite",
+        "agent_start_timeout_s": 30,
+        "env_params": {
+            "results_root": "/tmp/agentcompass-test-results",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/agentcompass-test-results"

(S108)